woothemes/woocommerce-rest-api-docs
# Add orders, orders stats, revenue stats analytics reports to API doc

Part of #248. This PR intends to merge into a feature branch [feature/add-wc-analytics-reports-api-doc](https://github.com/woocommerce/woocommerce-rest-api-docs/tree/feature/add-wc-analytics-reports-api-doc). But since the PR #248 hasn't been merged, the base branch of this PR is set to the `add/248-analytics-reports-products-api-doc` branch.

This PR adds orders, orders stats, and revenue stats APIs to the analytics doc:

- Add orders reports section (`/wc-analytics/reports/orders`)
- Add orders stats reports section (`/wc-analytics/reports/orders/stats`)
- Add revenue stats reports section (`/wc-analytics/reports/revenue/stats`)

#### Detailed test instruction

1. Build the doc by running `./build.sh`
2. Open `build/index.html`
3. Click `Analytics docs`
4. Verify the API docs under `Revenue Stats`, `Orders`, and `Orders Stats`.
